manifests/s/Swift/Toolchain/5.8/Swift.Toolchain.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-25267-1000.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.8
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+Scope: machine
+UpgradeBehavior: uninstallPrevious
```

Comment:
Yes, this is set intentionally.  The upgrade process doesn't always properly upgrade to use the of heat.  I hope that I can sort that out and make it possible to upgrade in place rather than uninstall and re-install.  Fortunately, there are no settings that need to be preserved by the compilers.